File: r_csv/save.R
# Function to save data to a CSV file
save_data <- function(data_frame, file_path) {
  # Create the directory if it doesn't exist
  dir.create(dirname(file_path), recursive = TRUE, showWarnings = FALSE)
  write.csv(data_frame, file_path, row.names = FALSE)
}

File: python_csv/save.py
import pandas as pd
import os
from io import StringIO

def save_data(table, file_path):
    # Create the directory if it does not exist
    os.makedirs(os.path.dirname(file_path), exist_ok=True)

    # Wrap HTML in StringIO object
    html_string = StringIO(str(table))

    # Convert table to DataFrame and save as CSV
    df = pd.read_html(html_string)[0]
    df.to_csv(file_path, index=False)